src/LevelEditor.cpp

```diff
@@ -1701,11 +1702,14 @@ std::string MoveGameObjectCommand::describe() {
 	if (objects.size() == 1) {
 		const bool isResize = oldPosition[0].w != newPosition[0].w || oldPosition[0].h != newPosition[0].h;
 		Scenery *scenery = dynamic_cast<Scenery*>(objects[0]);
-		return tfm::format(isResize ? _("Resize %s") : _("Move %s"), scenery ? (scenery->sceneryName_.empty() ? _("Custom scenery block")
-			: describeSceneryName(scenery->sceneryName_).c_str())
-			: _(blockNames[objects[0]->type]));
+		return tfm::format(isResize ? _("Resize %s") : _("Move %s"), scenery ?
+			/// TRANSLATORS: Context: Resize/Move ...

```

Comment:
I think this comment is also wrong, there is actually a scenery block named "Custom scenery block". Maybe you should added a comment two lines above?

Reply:
This string appears in multiple contexts, so we need to document each context - it's best done at each occurrence of the string, because xgettext will pick up all of them. Otherwise, the context comment will be too far away from its code IMO.